### DISCUSSION TITLE
missing 1 required keyword-only argument: 'intents

### DISCUSSION THREAD

**Discussion by imdjd** [Rapptz/discord.py#8422] [Q&A]
![image](https://user-images.githubusercontent.com/35808161/187710737-4cb168c2-e52d-4e76-963b-327d6d116a71.png)

can anyone help me i dont get it is it old or just out dated?

**Comment by Rapptz** [comment]
https://discordpy.readthedocs.io/en/stable/migrating.html#intents-are-now-required

It might help to read this page before updating.
